src/components/Header.js

```diff
@@ -8,54 +7,59 @@ import {
   NavItem,
   Collapse,
   Button,
-  Modal,
-  ModalHeader,
-  ModalBody,
-  FormGroup,
-  Label,
 } from 'reactstrap';
-import { Control, LocalForm, Errors } from 'react-redux-form';
-import { required, minLength, validEmail } from '../../validations/index';
+import { connect } from 'react-redux';
+import SignUser from './SignUser';
+import { postSignIn, postLogout } from '../stores/users/userActionCreators';
+
+const mapStateToProps = (state) => ({
+  user: state.user,
+});
+
+const mapDispatchToProps = (dispatch) => ({
+  postSignIn: (email, password) => dispatch(postSignIn(email, password)),
+  postLogout: (token) => dispatch(postLogout(token)),
+});
```

Comment:
It not affecting the code in any way, but in practice, you always see `mapStateToProps` and `mapDispatchToProps` after the component and not before